### PR TITLE
Enable Active-Active gateway feature flag

### PR DESCRIPTION
**Pull request by Nilambari**

### CODE REVIEW

File: arm-network/2016-06-01/swagger/network.json [hunk 1]
```diff
@@ -7643,7 +7647,7 @@
           "description": "Gets provisioning state of the VirtualNetworkGateway resource Updating/Deleting/Failed"
         }
       },
-      "description": "VirtualNeworkGateay properties"
+      "description": "VirtualNeworkGateway properties"
```
Comment (amarzavery) [inline arm-network/2016-06-01/swagger/network.json]: VirtualNe**t**workGateway , t is missing

File: arm-network/2016-06-01/swagger/network.json [hunk 2]
```diff
@@ -7618,6 +7618,10 @@
           "type": "boolean",
           "description": "EnableBgp Flag"
         },
+		"activeActive": {
```
Comment (tbombach) [inline arm-network/2016-06-01/swagger/network.json]: Formatting?

**Comment by tbombach** [comment]
Merged

**Comment by Nilambari** [comment]
@tbombach  Thanks a lot!

**Pull request merged**